system/modules/HTMLHelper/models/HTMLHelper.cfc

```diff
@@ -116,7 +116,7 @@ component extends="coldbox.system.FrameworkSupertype" accessors=true singleton{
 			} )
 			.each( function( item ){
 				// Load Asset
-				if( findNoCase( ".js", item ) ){
+				if( listLast( listFirst( listFirst( item, '##' ), '?' ), '.' ) EQ 'js' ){
```

Comment:
I fail to see how the `findNoCase()` wouldn't still find a `.js` in the path since it is an open search of the string.  I am not sure I follow the pull request, can you expand

Reply:
HI @lmajano can you try this gist:

    item = 'cdnsource/gridstack.js/style.min.css';
    isJs = findNoCase( ".js", item );
    isJs2 =  listLast( listFirst( listFirst( item, '##' ), '?' ), '.' ) EQ 'js';
    writedump(isJs);
    writedump(isJs2);

Reply:
I totally get it now.  Tricky.  TRhanks